imports/client/components/ProfileList.tsx

```diff
@@ -277,12 +277,12 @@ const ProfileList = ({
     const isInteresting = (user: Meteor.User) => {
       for (let i = 0; i < toMatch.length; i++) {
         const searchKey = toMatch[i];
-        if (user.profile?.displayName?.toLowerCase().indexOf(searchKey) === -1 &&
+        if (user.displayName?.toLowerCase().indexOf(searchKey) === -1 &&
           user.emails?.every((e) => (
```

Comment:
Once we switch to subscribing to only `avatars` and `displayNames`; user.emails will always be empty, as will user.phoneNumber (and maybe `user.roles` too?).  Being unable to search over those fields feels like a functional regression?